apps/docs/src/pages/components/Playground.tsx

```diff
@@ -149,13 +227,13 @@ export const Playground = ({
 
   return (
     <>
-      <div className="flex justify-between p-2 mt-1 border border-[var(--uikit-colors-atmo4)] rounded-t-round">
-        <div className="w-[70%] flex justify-center items-center">
+      <div className="flex justify-between mt-1 border border-[var(--uikit-colors-atmo4)] rounded-t-round">
+        <div className="w-[70%] flex justify-center items-center p-[var(--uikit-space-sm)]">
           <Component {...propsState} {...componentProps}>
             {children?.props.children}
           </Component>
         </div>
-        <div className="w-[30%] flex flex-col gap-[var(--uikit-space-xs)] justify-center items-center border-l border-[var(--uikit-colors-atmo3)] pl-[var(--uikit-space-sm)]">
+        <div className="w-[30%] flex flex-col gap-[var(--uikit-space-xs)] justify-center items-center border-l border-[var(--uikit-colors-atmo3)] p-[var(--uikit-space-sm)] pl-[var(--uikit-space-xs)]">
```

Comment:
Don't the breakpoints work? This is really strange
```suggestion
        <div className="w-[30%] flex flex-col gap-xs justify-center items-center border-l border-[var(--uikit-colors-atmo3)] p-sm pl-xs">
```